_i18n/en.yml

```diff
@@ -599,24 +605,46 @@ policies:
     or [contact us](site.baseurl/contact).
 help_pages:
   changing-settings:
-    how-do-i-change-my-email-address: To change your email address, select Edit next
-      to the email address. Enter the new email address and submit it. Then check
-      your email account and look for an email from login.gov. Open the email and
-      click on the link provided. Clicking the link will redirect you back to login.gov.
-      Once you reach login.gov your new email address is established.
+    how-do-i-change-my-email-address: To change the email associated with your login.gov account, go to [https://www.login.gov/](https://www.login.gov/) and click “Manage Account” to sign into your login.gov account. Select “edit” next to the email address. Enter the new email address and submit it. Then check your email account and look for an email from login.gov. Open the email and click on the link provided. Clicking the link will redirect you back to login.gov. Once you reach login.gov, your new email address is established.
```

Comment:
In general, we shouldn't be asking folks to click things. Select, choose, follow are all good verbs. Click implies a mouse which is not a safe assumption.

Reply:
@clantosswett ☝️

Reply:
Let's go with select